src/_cffi_src/build_openssl.py

```diff
@@ -117,13 +116,6 @@ def _extra_compile_args(platform):
         "callbacks",
     ],
     libraries=_get_openssl_libraries(sys.platform),
-    # These args are passed here so that we only do Wconversion checks on the
```

Comment:
Why remove this? It's still true that moving these args into CFLAGS would result in failures due to cffi generated code right?

Reply:
Because the comment is duplicated, and I didn't want to have to update it 2x every time.